woocommerce/woocommerce-android
# [Simplified login] Site credentials step for native jetpack installation

Closes: #7796 


### Description
This PR adds the site credentials step for the native Jetpack installation, the work is based on what we've added previously for the account mismatch (the PR #7471), I didn't try to make the code reusable, I just copied it, as we will be removing the feature from the account mismatch after the end of the experiment.

@JorgeMucientes I'm requesting your review alone, as you reviewed #7471, and this PR is basically a copy paste of those changes, thanks in advance 🙏.

### Testing instructions
1. Repeat steps of #7850, the click on the primary button
2. Confirm the UI is correct for both installation and connection flows.
3. Enter username/password and confirm they work as expected.
4. Try signing using  a wrong username/password.
5. Confirm an error message is shown.
6. Try using the correct username/password.
7. Confirm a Toast is shown with the text TODO.

### Images/gif
| Install | Connect |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/1657201/202162443-78adb3cf-2b8c-48a3-b753-0a12e4847708.png) | ![image](https://user-images.githubusercontent.com/1657201/202162653-0168eb7f-1ffd-446b-bda5-26dbed807994.png) |
